src/screens/manage-account/manage-account.tsx

```diff
@@ -71,21 +80,39 @@ export class ManageAccountComponent extends React.Component<
     INavigationProps & IReduxProps & IThemeProps<ReturnType<typeof stylesProvider>>
 > {
     public static navigationOptions = navigationOptions;
-    public accountsSwipeableRef: ReadonlyArray<string> = new Array();
+    public accountsSwipeableRef: any = [];
     public currentlyOpenSwipeable: string = null;
 
     public componentDidMount() {
-        this.props.navigation.setParams({
-            blockchain: this.props.selectedAccount.blockchain
-        });
+        this.props.navigation.setParams({ blockchain: this.props.selectedAccount.blockchain });
+        this.showHints();
+    }
+
+    private showHints() {
+        const tokens = Object.keys(this.props.selectedAccount.tokens);
+        if (
+            tokens &&
+            tokens[1] &&
+            this.props.hints.MANAGE_ACCOUNT.TOKENS_LIST < DISPLAY_HINTS_TIMES
+        ) {
+            const id = tokens[1];
+
+            setTimeout(() => {
```

Comment:
we may want to trigger hints display from other places, i would recommend to not use `setTimeout` here. Use it in `componentDidMount`

Reply:
it's applicable for all occurrences :)